tgui/packages/tgui/interfaces/ShuttleControlConsole.tsx

```diff
@@ -0,0 +1,99 @@
+import { useBackend } from '../backend';
+import { Button, Section, Box, LabeledList } from '../components';
+import { Window } from '../layouts';
+
+export type ShuttleControlConsoleData = {
+  shuttle_status: string;
+  shuttle_state: string;
+  has_docking: boolean;
+  docking_status: string;
+  docking_override: boolean;
+  can_launch: boolean;
+  can_cancel: boolean;
+  can_force: boolean;
+  can_rename_ship: boolean;
+  ship_name: string;
+};
+
+export const ShuttleControlConsole = (props, context) => {
+  const { act, data } = useBackend<ShuttleControlConsoleData>(context);
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <Section title="Shuttle Status">
+          <Box>{data.shuttle_status}</Box>
+          <Box>Ship designation and name: {data.ship_name}</Box>
+          {data.can_rename_ship ? (
+            <Box>
+              <Button
+                content="Rename"
+                icon="pen"
+                onClick={() => act('rename')}
+              />
+            </Box>
+          ) : null}
+          <Box>
+            <LabeledList>
+              <LabeledList.Item label="Engines">
+                {data.shuttle_state === 'idle' ? (
+                  <Box>Idle</Box>
+                ) : data.shuttle_state === 'warmup' ? (
+                  <Box>Starting Ignition</Box>
+                ) : data.shuttle_state === 'in_transit' ? (
+                  <Box>Engaged</Box>
+                ) : (
+                  <Box>Error</Box>
+                )}
+              </LabeledList.Item>
+              {data.has_docking ? (
+                <LabeledList.Item label="Docking Status">
+                  {data.docking_status === 'docked' ? (
+                    <Box>Docked</Box>
+                  ) : data.docking_status === 'docking' ? (
+                    data.docking_override ? (
+                      <Box>Docking-Manual</Box>
```

Comment:
You don't need to use empty boxes for text. You should only use them if you're altering the text in some way or you need the positioning to be specific (not the case here since it's in a labeled list).
Remove all the <Box>es that don't alter text